webpack/JobWizard/JobWizard.js

```diff
@@ -116,23 +126,32 @@ export const JobWizard = () => {
           jobTemplateID={jobTemplateID}
         />
       ),
-      canJumpTo: isTemplate,
+      canJumpTo: isTemplate && valid.hostsAndInputs,
+      enableNext: isTemplate && valid.hostsAndInputs && valid.advanced,
     },
     {
       name: WIZARD_TITLES.schedule,
       component: (
         <Schedule
           scheduleValue={scheduleValue}
           setScheduleValue={setScheduleValue}
+          setValid={newValue => {
+            setValid(currentValid => ({ ...currentValid, schedule: newValue }));
+          }}
         />
       ),
-      canJumpTo: isTemplate,
+      canJumpTo: isTemplate && valid.hostsAndInputs && valid.advanced,
+      enableNext:
+        isTemplate && valid.hostsAndInputs && valid.advanced && valid.schedule,
     },
     {
       name: WIZARD_TITLES.review,
       component: <p>Review Details</p>,
       nextButtonText: 'Run',
-      canJumpTo: isTemplate,
+      canJumpTo:
+        isTemplate && valid.hostsAndInputs && valid.advanced && valid.schedule,
+      enableNext:
+        isTemplate && valid.hostsAndInputs && valid.advanced && valid.schedule,
```

Comment:
same here, I think that for clicking next / run we can check just for the validity of the step itself